stonesoup/dataassociator/probability.py

```diff
@@ -52,14 +46,18 @@ def associate(self, tracks, detections, time):
             track: self.hypothesiser.hypothesise(track, detections, time)
             for track in tracks}
 
-        return associate_highest_probability_hypotheses(tracks, hypotheses)
+        # Ensure association probabilities are normalised
+        for track, hypothesis in hypotheses.items():
+            hypothesis.normalise_probabilities(total_weight=1)
+
```

Comment:
This normalisation is not currently necessary, given that multi-hypotheses are already normalised on instantiation in PDAHypothesiser and (J)PDA requires the use of PDAHypothesiser. However, PDA should always produce normalised weights, whereas normalising weights is not technically necessary inside PDAHypothesiser. Therefore, I have included this normalisation to make sure normalisation is satisfied, in case users define some custom hypothesiser that does not normalise.